### PR TITLE
Fix libmnl-devel not found in 7.8

### PR DESCRIPTION
**Pull request by LiliDeng**
Test Results -

```
[LISAv2 Test Results Summary]
Test Run On           : 05/11/2020 06:41:49
ARM Image Under Test  : Redhat : RHEL : 7.8 : latest
Initial Kernel Version: 3.10.0-1127.el7.x86_64
Final Kernel Version  : 3.10.0-1127.el7.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:28

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 DPDK                 VERIFY-DPDK-FAILSAFE-DURING-TRAFFIC                                               PASS                23.54 
dpdk_version test_mode phase   fwdrx_pps_avg fwdtx_pps_avg
------------ --------- -----   ------------- -------------
18.11.0      txonly    after   4371574       4371575      
18.11.0      txonly    before  4632470       4632470      
18.11.0      txonly    during  68697         68697        
18.11.0      txonly    overall 2426233       2426233 

[LISAv2 Test Results Summary]
Test Run On           : 05/11/2020 04:30:50
ARM Image Under Test  : Redhat : RHEL : 7.8 : latest
Initial Kernel Version: 3.10.0-1127.el7.x86_64
Final Kernel Version  : 3.10.0-1127.el7.x86_64
Total Test Cases      : 2 (2 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:11

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 DPDK                 VERIFY-DPDK-COMPLIANCE                                                            PASS                  6.2 
    2 DPDK                 VERIFY-DPDK-RING-LATENCY                                                          PASS                 4.86 
```

### CODE REVIEW

**Comment by LiliDeng** [comment]
Regression Test -
```
[LISAv2 Test Results Summary]
Test Run On           : 05/11/2020 07:15:36
ARM Image Under Test  : Redhat : RHEL : 7.6 : latest
Initial Kernel Version: 3.10.0-957.35.2.el7.x86_64
Final Kernel Version  : 3.10.0-957.35.2.el7.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:13

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 DPDK                 VERIFY-DPDK-COMPLIANCE                                                            PASS                 9.65 
```

File: Testscripts/Linux/dpdkUtils.sh [hunk 1]
```diff
@@ -167,8 +167,9 @@ function Install_Dpdk () {
 			ssh "${1}" ". utils.sh && install_epel"
 			ssh "${1}" "yum -y groupinstall 'Infiniband Support' && dracut --add-drivers 'mlx4_en mlx4_ib mlx5_ib' -f && systemctl enable rdma"
 			check_exit_status "Install Infiniband Support on ${1}" "exit"
-			ssh "${1}" "(grep 7.5 /etc/redhat-release || grep 7.6 /etc/redhat-release) && curl https://partnerpipelineshare.blob.core.windows.net/kernel-devel-rpms/CentOS-Vault.repo > /etc/yum.repos.d/CentOS-Vault.repo"
+			ssh "${1}" "(grep -E '7.5|7.6|7.8' /etc/redhat-release) && curl https://partnerpipelineshare.blob.core.windows.net/kernel-devel-rpms/CentOS-Vault.repo > /etc/yum.repos.d/CentOS-Vault.repo"
```
Comment (harz566) [inline Testscripts/Linux/dpdkUtils.sh:170]: what about 7.x that's newer than 7.8 in the future? should we change step by step? any inconvenience if we place 7.[\d+] here?

Comment (LiliDeng) [reply Testscripts/Linux/dpdkUtils.sh:170]: 7.7, 8.0, 8.1 and 8.2 it has libmnl-devel package in default repository.

Comment (juhlee-microsoft) [reply Testscripts/Linux/dpdkUtils.sh:170]: Thanks, @LiliDeng This kinds of dependency repo change has been headache, I wished those were consistent, but I believe each distro repo management has new plan. So we should check the same problem in every new version release. Thanks for taking care of it.

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**